lib/idx/types/index.ts

```diff
@@ -59,6 +59,7 @@ export type NextStep = {
   canResend?: boolean;
   inputs?: Input[];
   options?: IdxOption[];
+  pollForResult?: IdxPollOptions;
```

Comment:
How about name it as `poll` (feels like better future proof), also add a `required: boolean;` field in it, so devs can get hint about if calling idx.poll is optional or not. For Okta verify scenario, `required` should be true, but for challenge-authenticator step, it can be false.